modules/tinymce/src/core/main/ts/api/html/SaxParser.ts

```diff
@@ -118,40 +118,67 @@ const isInvalidUri = (settings: SaxParserSettings, uri: string, tagName: string)
   }
 };
 
+interface LightweightTag {
```

Comment:
Anyone want to help me find better names for... all of this? I'd prefer to have not split this out into a function in the first place, because it's really just an implementation detail for `findMatchingEndTagIndex`, but unfortunately JS won't let me assign to the results of loops or complex if statements.